data/json/items/ammo/20x66mm.json

```diff
@@ -102,7 +102,7 @@
     "id": "20x66_inc",
     "copy-from": "20x66_shot",
     "type": "AMMO",
-    "name": { "str": "20x66mm incendiary", "str_pl": "20x66mm incendiaries" },
+    "name": { "str_sp": "20x66mm incendiary" },
```

Comment:
The problem here being that "20x66mm incendiaries" means more than one different type of incendiary round (incendiary is an adjective, not a noun....).

Reply:
*Is* there more than one type of _20x66_ incendiary, though?

Reply:
If someone tosses you an incendiary grenade and you ask what it is, "An incendiary" is a grammatically-acceptable response, since "grenade" or "device" is implied by context.

Reply:
Point.

Reply:
Reverted to original.

Reply:
I just realized - these are used in a list (that includes something indicating rounds, 20x66mm).